web/src/components/MapPopup.tsx

```diff
@@ -0,0 +1,31 @@
+import { GeoJsonProperties, Position } from 'geojson'
+import React, { ReactElement } from 'react'
+import { Popup } from 'react-map-gl'
+import styled from 'styled-components'
+
+const StyledPopup = styled(Popup)`
+  width: 250px;
+`
+
+interface MapPopupProps {
+  coordinates: Position
+  properties: GeoJsonProperties
+}
+
+const MapPopup: React.FC<MapPopupProps> = (props: MapPopupProps): ReactElement => {
+  const { coordinates, properties } = props
+  return (
+    <StyledPopup
+      longitude={coordinates[0]}
+      latitude={coordinates[1]}
+      closeButton={false}
+      closeOnClick={false}
+      anchor='bottom'>
+      <div>
+        <strong>{properties?.title}</strong>
```

Comment:
Wouldn't it make more sense to just not show a popup if there is no title? Not sure about that or rather whether it is possible at all that there is no title.

Reply:
i mean there is an extra popup ticket where we define whats shown there and what not. it was just to prove right thats its the right popup connected to the marker